### ISSUE TITLE
batch delete resources ( proguard, sourcemaps...)

### ISSUE THREAD

**Issue by farfromrefug** [getsentry/sentry#73878]
### Problem Statement

Often i want to cleanup old releases sourcemaps/proguard mappings to save up storage.
Right now i have to do it one by one, and that for each of my project.

### Solution Brainstorm

add the possibility to select some/all resources and then delete them. Exactly the same way as you can do it with issues.

### Product Area

Unknown

**Comment by chargome** [comment]
@Swatinem, do we have plans to add batch delete?

**Comment by farfromrefug** [comment]
@Swatinem @chargome sorry to bump this but this is really import to have something like this.
One issue with "debug files" is server storage. Those files can be huge (android, ios) and use a lot of storage.
beeing able to easily remove them can help cleaning space and save storage.
So i will push that request further as it is getting really important (right now i have to stop uploading debug data because of this!):
* allow to select multiple items and call an action (delete in this case)
* maybe allow to filter by date (older than a year for example) to easily clear old data
* the best would even be a project setting to automatically remove old debug data (like a year old)

**Comment by mydea** [comment]
cc @obostjancic did some improvements in this space recently. Not sure if this is addressed there or if we _could_ address it, Ogi do you have thoughts on this?

**Comment by jjbayer** [comment]
@matejminar is this something your team could look into?

**Comment by farfromrefug** [comment]
@matejminar really ? I guess this is optional. 
Still think we Gould be able to batch remove. Like after bad release or something

**Comment by lforst** [comment]
> My understanding is that these files are now being automatically cleaned up after 90 days of the last use.

Can confirm.

To be fully transparent: This feature is not a priority for us. For bad releases you can already delete all the sourcemaps associated with a particular upload by going to "Source Maps" in your project settings.

**Comment by farfromrefug** [comment]
@lforst yes you can but that exactly the problem . you have to do.it one by one , same for all.upload files like mappings
